neuro_extras/seldon.py

```diff
@@ -96,6 +96,7 @@ async def _create_seldon_deployment(
                 "name": "neuro-download",
                 "image": NEURO_EXTRAS_IMAGE,
                 "imagePullPolicy": "Always",
+                "securityContext": {"runAsUser": 0},
                 "command": ["bash", "-c"],
                 "args": [
                     "cp -L -r /var/run/neuro/config /root/.neuro;"
```

Comment:
actually, with the new authentication method this one might be broken, we need to check it. Anyways, Seldon deployment will evolve a lot soon.

Reply:
That's true, the only moto to fix this is to resolve the current state, but we will replace this functionality with [kube-gateway](https://github.com/neuro-inc/mlops-kube-gateway).